pkg/ebpf/events_definitions.go

```diff
@@ -6322,7 +6322,7 @@ var EventsDefinitions = map[int32]EventDefinition{
 		},
 		Sets: []string{},
 		Params: []trace.ArgMeta{
-			{Type: "HookedSyscallData[]", Name: "hooked_syscalls"},
+			{Type: "bufferdecoder.HookedSyscallData[]", Name: "hooked_syscalls"},
```

Comment:
I can't agree with what you have done here.
If you want to export a type through signature, it should be under the `types` package.